DemoApp-Swift/DemoApp-Swift/ALAppDelegate.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ALAppDelegate.swift
+//  DemoApp-Swift
+//
+//  Created by Andrew Tian on 9/20/19.
+//  Copyright © 2019 AppLovin. All rights reserved.
+//
+
+import UIKit
+import AppLovinSDK
+
+class ALAppDelegate: UIResponder, UIApplicationDelegate
+{
+    var window: UIWindow?
+    
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool
+    {
+        #warning("Make sure to add your AppLovin SDK key in the Info.plist under the \"AppLovinSDKKey\" key")
+        
+        // Initialize the AppLovin SDK
+        ALSdk.shared()?.mediationProvider = ALMediationProviderMAX;
```

Comment:
Let's do force unwrap here and L22 - `shared` returns `nil` if the publisher forgets to include the SDK key in the Info.plist. So we'll crash if it's not there.